sdk/dynamicconfig/example/README.md

```diff
@@ -1,41 +1,72 @@
-# Dynamic config service example
+# Dynamic Metric Configuration
+This example showcases the ability to configure metric collection periods at
+runtime, via a remote configuration service. It is a prototype implementation of
+[this experimental
+specification](https://github.com/open-telemetry/opentelemetry-specification/blob/master/experimental/metrics/config-service.md). The prototype is currently available
+only for Go.
 
-Right now the generated protobuf .go file we're using is in our fork of the repo. Eventually we want to import in from opentelemetry-proto.
+**Note: this system is experimental, and liable to rapid breaking change.
+Use at your own risk.**
```

Comment:
Let's remove ", and liable to rapid breaking change."

Reply:
removed